ops/gulp/docs-build.js

```diff
@@ -5,7 +5,7 @@ let environment = require('../lib/environment');
 
 const task = (cb) => {
 
-    gulpSequence('pkg-build', 'docs-clean', ['docs-resources', 'docs-html', 'docs-css', 'docs-styleguide', 'docs-js'], cb);
+    gulpSequence('pkg-build', 'docs-clean', ['docs-resources', 'docs-html', 'docs-css', 'docs-styleguide', 'docs-js', 'pkg-icons'], cb);
```

Comment:
`pkg-icons` shouldn't be called separately. It should be part of the `pkg-build` process that runs before any docs tasks even begin. As mentioned below, if you need a `docs-icons` task to reach into `src` and grab the icons sprite then create a separate one for just that.